src/ResourceManager/Advisor/Commands.Advisor/help/Get-AzAdvisorConfiguration.md

```diff
@@ -0,0 +1,101 @@
+---
+external help file: Microsoft.Azure.Commands.Advisor.dll-Help.xml
+Module Name: Az.Advisor
+online version: https://docs.microsoft.com/en-us/powershell/module/az.advisor/get-azadvisorconfiguration
+schema: 2.0.0
+---
+
+# Get-AzAdvisorConfiguration
+
+## SYNOPSIS
+Get the Azure Advisor configurations for the given subscription or resource group.
+
+## SYNTAX
+
+```
+Get-AzAdvisorConfiguration [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The configurations associated with a subscription have two types:
+
+ResourceGroup level configuration: There can be only one configuration for each ResourceGroup in a subscription. Exclude is the only property of this type of configuration.
+
+ResourceGroup level configuration: There can be only one configration for each ResourceGroup in a subscription, only exclude property is taken into effect for configuration.
```

Comment:
*configuration

I still am not sure what "taken into effect" means - does it mean it's the only property you can set? The only property that is displayed? Please explain what this means.

Reply:
I updated that, should be insync with other docs that you suggested. 

Takes into effect means - only that property can be even set, updated and will be applied during the configuration process. Once that configuration is set say, LowCpuThreshold of 15 will be applied to all the recommendations.